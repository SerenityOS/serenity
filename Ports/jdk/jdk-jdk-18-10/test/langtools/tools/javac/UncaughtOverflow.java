/*
 * @test /nodynamiccopyright/
 * @bug 4035346 4097402
 * @summary Compiler used to allow this initialization, despite the overflow.
 * @author turnidge
 *
 * @compile/fail/ref=UncaughtOverflow.out -XDrawDiagnostics UncaughtOverflow.java
 */

public
class UncaughtOverflow {
    int i1 = 100000000000;
    int i2 = -2147483649;
}
