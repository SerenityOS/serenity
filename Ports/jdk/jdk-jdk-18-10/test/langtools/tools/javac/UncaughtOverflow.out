UncaughtOverflow.java:12:14: compiler.err.int.number.too.large: 100000000000
UncaughtOverflow.java:13:15: compiler.err.int.number.too.large: -2147483649
2 errors
